### PR TITLE
Proposal standard table

### PR DESCRIPTION
**Pull request by gianarb**
Hi! This is a proposal to create a standard schema for all component.. ( thanks Doctrine :P ).. 
We can use it in other components.. And for example we can add another column 3.0 to check status of ZF3 work..

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -358,3 +361,13 @@ At times, it's useful to either create a string representation of a message (ser
...
+
+
+  [Master]: https://travis-ci.org/zendframework/zend-diactoros
+  [Master image]: https://secure.travis-ci.org/zendframework/zend-diactoros.svg?branch=master
+  [Master coverage image]: https://img.shields.io/coveralls/zendframework/zend-diactoros/master.svg
+  [Master coverage]: https://coveralls.io/r/zendframework/zend-diactoros?branch=master
+  [Develop]: https://github.com/zendframeowork/zend-diactoros/tree/develop
+  [Develop image]:  https://secure.travis-ci.org/zendframework/zend-diactoros.svg?branch=develop
+  [Develop coverage image]: https://coveralls.io/repos/zendframework/zend-diactoros/badge.svg?branch=develop
+  [Develop coverage]: https://coveralls.io/r/zendframework/zend-progressbar?branch=develop
```
Comment (Maks3w) [inline README.md]: progressbar?

Comment (gianarb) [inline README.md]: Sorry :) replace

**Comment by Maks3w** [comment]
I prefer the ZF2 inline style.

If the repo need scale to more branches (like per version branch) this have sense, but actually the maximum number of branches will be 2.

**Comment by gianarb** [comment]
In my opinion all other components, Cache, Config, Mvc will require other
branches :)

2015-05-26 9:18 GMT+02:00 Maks3w <notifications@github.com>:

> I prefer the ZF2 inline style.
>
> If the repo need scale to more branches (like per version branch) this
> have sense, but actually the maximum number of branches will be 2.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendframework/zend-diactoros/pull/18#issuecomment-105422288>
> .
>



-- 
Gianluca Arbezzano
www.gianarb.it

**Comment by samsonasik** [comment]
@gianarb I am +1 for other branches so we can exactly have branches : 2.4, 2.5, 3.0, etc so imo it be easier

**Comment by Maks3w** [comment]
@samsonasik Please, define "easier". 

Merge a PR across all branches requires a lot git operations and deal with conflicts.
People have problems when target the PR between master and develop and now there are more choices for to have PRs targetting wrong branches.

Finally this is not the place for to discuss how the project should be.

**Comment by samsonasik** [comment]
easier to target of course, bug fix from 2.4 should be to 2.4 branch, feature 2.5 for 2.5, 3.0 for major bc break

**Comment by gianarb** [comment]
@Maks3w maybe this isn't a good place to discuss about this stuff but mailing list is dead :skull: 

I don't know the future of ZF dev flow but I'm agree with @samsonasik in this moment.. It's easier and clear..

**Comment by Ocramius** [comment]
I'll merge this for now - further improvements = further PRs.

At least we now have badges for all branches.

**Comment by Ocramius** [comment]
Merged, thanks!

`master`: zendframework/zend-diactoros@c6f4d6342e585d39b88a9529d910acb3aa8e67bc
`develop`: zendframework/zend-diactoros@8d580736bc9e3b110dc1d377774fc60353870cb0

**Pull request closed without merging**